### ISSUE TITLE
Some of the test items do not have squad attribute reported in XML junit file and then not in RP 

### ISSUE THREAD

**Issue by petr-balogh** [red-hat-storage/ocs-ci#6075]
For example here:
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/j-240vu1cs33-ua/j-240vu1cs33-ua_20220621T051522/logs/test_results_1655788203.xml

I don't see any Squad attribute added for example this test case:
tests.test_failure_propagator.TestFailurePropagator
or
tests.ui.test_pvc_ui.TestPvcUserInterface

**Comment by clacroix12** [comment]
`tests.test_failure_propagator.TestFailurePropagator` is at the top level of the `tests` directory and therefore doesn't have a default squad marker assigned in the `SQUADS` constant. It also isn't decorated with a squad, so this doesn't appear to be a bug. If we want a test marker we should move this test to another appropriate location that has a squad assigned, or add a decorator to specify the squad.

`tests.ui.test_pvc_ui.TestPvcUserInterface` is decorated with a squad market at the class level but it is also parametrized which I think might be causing the issue here. I haven't pinned down the root cause yet but I'm testing a few things out regarding the test markers and parametrized tests. I will report back as soon as I have something.

**Comment by clacroix12** [comment]
Another issue I'm seeing is that some test cases have multiple squad markers.

```
<testcase classname="tests.manage.mcg.ui.test_mcg_ui.TestStoreUserInterface" name="test_store_creation_and_deletion[namespacestore]" file="tests/manage/mcg/ui/test_mcg_ui.py" line="39" time="29.443">
<properties>
<property name="squad" value="Red"/>
<property name="squad" value="Black"/>
<property name="polarion-testcase-id" value="OCS-2547"/>
</properties>
</testcase>
```

In ReportPortal however, they only have one of the squad markers since we are moving on once we find one of them.
![Screen Shot 2022-07-26 at 11 02 50 AM](https://user-images.githubusercontent.com/1605721/181079051-f70c7d0b-4fe4-4751-ab27-467b87bd5ae5.png)

This is due to the filepaths we define in the `SQUADS` both being in the path for this test case (and several others).

```
SQUADS = {
    ...
    "Red": ["/mcg/", "/rgw/"],
    ...
    "Black": ["/ui/"],
    ...
}
```

This should get resolved when we change how we are assigning ownership, but I wanted to call it out now that I'm noticing it.

**Comment by clacroix12** [comment]
I believe I have found the underlying issue here. I wrote a few brand new tests and decorated them with squad markers at every supported location (function, method, class, and parametrized fixture) and noticed that none of them ended up with squad attributes in the junit report. It wasn't until I looked at where we are applying the squad markers when I realized that we aren't actually doing anything with those squad markers:

https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L188-L189

If the marker exists, we are continuing on to where we apply the filepath based markers as well as adding the item attribute. When I changed the logic here to add the attribute as well for existing markers all of the decorated squad markers appeared in the junit report properly.

To address this I have raised #6233
